benchmarks/views/index.py

```diff
@@ -35,29 +36,29 @@ def view(request):
 
 
 def get_context(user=None):
-    benchmarks = _collect_benchmarks()
+    benchmarks = _collect_benchmarks(with_versions=True if user is not None else False)
     model_rows = _collect_models(benchmarks, user)
 
     # to save vertical space, we strip the lab name in front of benchmarks.
     uniform_benchmarks = {}  # keeps the original benchmark name
     for benchmark in benchmarks:  # remove lab for more compactness
-        benchmark.long_name = benchmark.benchmark_type.identifier
         uniform_benchmarks[benchmark.benchmark_type.identifier] = benchmark.benchmark_type.identifier
         match = re.match(r'[^\.]+\.(.+)', benchmark.benchmark_type.identifier)
         if match:
             uniform_benchmarks[benchmark.benchmark_type.identifier] = match.group(1)
-            benchmark.identifier = benchmark.benchmark_type.identifier = match.group(1)
+            benchmark.short_name= benchmark.benchmark_type.identifier = match.group(1)
         else:
-            benchmark.identifier = benchmark.benchmark_type.identifier
+            benchmark.short_name = benchmark.benchmark_type.identifier
         benchmark.ceiling = represent(benchmark.ceiling)
+        benchmark.identifier = f'{benchmark.identifier}_v{benchmark.version}'
     # map from a benchmark to its parent
     benchmark_parents = {
-        benchmark.long_name: benchmark.benchmark_type.parent.identifier if benchmark.benchmark_type.parent else None
+        benchmark.identifier: f'{benchmark.benchmark_type.parent.identifier}_v0' if benchmark.benchmark_type.parent else None
```

Comment:
can you add a comment to explain the `_v0`? My guess is that it's adding a version number but why is it always 0?

Reply:
Added an explanation